rewrite-java/src/main/java/org/openrewrite/java/Internals.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import java.nio.file.Path;
+import java.util.Collection;
+
+public class Internals {
+    public static Collection<Path> getClasspath(JvmParser.Builder<?, ?> parserBuilder) {
+        return parserBuilder.classpath;
```

Comment:
I realize we could get rid of this if we move the code which updates the classpath from `JavaTemplateParser` to `JavaTemplate`, as this class is in the same package as `JvmParser`.

Reply:
There is already an `internal` package in `rewrite-java` for things like this: 
<img width="292" alt="image" src="https://github.com/openrewrite/rewrite/assets/1697736/0a41df0e-fefc-4701-a0f4-79de89d6e8d3">

Reply:
Yes, but JavaParser is not in that package. But like I pointed out, we can solve this without any additional accessor, since JavaTemplate is in the same package as JavaParser.